pyproject.toml

```diff
@@ -34,10 +34,11 @@ dependencies = [
   "PyYAML",
   "confluent_kafka",
   "ess-streaming-data-types",
-  "essreduce>=25.02.1",
+  "essreduce>=25.02.5",
   "jinja2",
   "pydantic>=2",
   "scipp>=25.02.0",  # Improved performance of scipp.group
+  "pooch",
```

Comment:
Why is pooch a dependency?

Reply:
Because the `detector_data` services pull geometry files via pooch.